src/main/java/uk/gov/hmcts/reform/civil/model/UnavailableDateLRspec.java

```diff
@@ -11,7 +11,7 @@
 @Data
 @Builder
 @IsPresentOrEqualToOrLessThanOneYearInTheFuture(groups = UnavailableDateGroup.class)
-public class SmallClaimUnavailableDate {
+public class UnavailableDateLRspec {
 
     private final String who;
     private final LocalDate date;
```

Comment:
Can we not refactor SmallClaimUnavailableDateType to make it UnavailableDateType just like we did for SmallClaimUnavailableDate

Reply:
I have used jaber's constant class which he has used for LRspec constants, and added my SINGLE_DATE and DATE_RANGE to that.